missed-moment.py

```diff
@@ -11,12 +11,15 @@
 from gpiozero import Button
 import picamera
 
+import constants
+
 # TODO from export.slack import slack
 
-MEDIA_DIR = '/missed_moment_media'
-TIME_TO_RECORD = 30  # in seconds
-AUDIO_CAPTURE_REMOTE_PORT = 7777
-AUDIO_CAPTURE_TEMP_FILENAME = f'{MEDIA_DIR}/mm-timemachine.wav'
+
+MEDIA_DIR = constants.MEDIA_DIR
+TIME_TO_RECORD = int(constants.TIME_TO_RECORD)
+AUDIO_CAPTURE_REMOTE_PORT = int(constants.AUDIO_CAPTURE_REMOTE_PORT)
+AUDIO_CAPTURE_TEMP_FILENAME = constants.AUDIO_CAPTURE_TEMP_FILENAME
```

Comment:
casting to `int()` is unnecessary here.

Reply:
As with above, these can all just be used directly.

Reply:
I removed the unnecessary casting, that was leftover from when I was trying .ini and .toml files for the config file.
I changed so the import statement directly imports the necessary variables.